src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -306,3 +310,241 @@ def test_WINNF_FT_S_MCP_1(self, config_filename):
+
+      # Step 11 : Send heartbeat request
+      domain_proxy_object.heartbeatForAllActiveGrants()
+
+      # Get the list of CBSDs
+      cbsds_with_domain_proxies.append(domain_proxy_object.getCbsdsWithAtLeastOneAuthorizedGrant())
+
+    # Register individual Cbsds(Domain Proxy with single CBSD)
+    for cbsd_record in iteration_content['cbsdRecords']:
+      proxy = test_harness_objects.DomainProxy(cbsd_record['clientCert'], cbsd_record['clientKey'], self)
+      proxy.registerCbsdsAndRequestGrants([cbsd_record['registrationRequest']], [cbsd_record['grantRequest']])
+      cbsd_as_dp_list.append(proxy)
+
+    # Step 12 : Invoke Aggregate Interference Model
+    # TODO: To invoke Aggregate Interference Model
+
+    # Check MCP.1
+    # TODO: To invoke checkMcpIap method
+
+    # Step 13 : Configure SAS Harness with FAD,trigger FAD generation
+    for test_harness, test_harness_data in zip(sas_test_harness_objects, iteration_content['sasTestHarnessData']):
+      sas_test_harness_dump_records = [test_harness_data['cbsdRecords']]
+      test_harness.writeFadRecords(sas_test_harness_dump_records)
+
+    # Step 14,15 : Trigger Full Activty Dump and Pull FAD records from SAS UUT
+    fad_uut_object = getFullActivityDumpSasUut(self._sas,
+                                               self._sas_admin)
+
+    # Step 16 : Trigger CPAS and wait for its completion
+    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
+
+    # Step 17 : Call IAP reference model
+    # TODO: To invoke DPA Move Reference model
+
+    # Call Pre IAP model
+    # TODO: To invoke Pre-iap reference model
+
+    # Call IAP reference model
+    # TODO: To invoke IAP reference model
+
+    # Step 18,19,20 and 21 :
+    # Send heartbeat request for the grants, relinquish the grant, grant request and heartbeat for new grant
+    for domain_proxy_object in domain_proxy_objects:
+      domain_proxy_object.performHeartbeatAndUpdateGrants()
+
+      # Step 22: Calculating the Aggregate interface and IAP reference model invoke.
+      cbsds_with_domain_proxies.append(domain_proxy_object.getCbsdsWithAtLeastOneAuthorizedGrant())
+
+    # Invoke Aggregate Interference Model
+    # TODO: To invoke Aggregate Interference Model
+
+    # Check MCP.1
+    # TODO: To invoke checkMcpIap method
+    # Execute from Step23 to Step30 only for MCP testcase
+    if test_type == 'MCP':
+      self.executeMCPOnlyTestSteps(iteration_content, domain_proxy_objects, cbsd_as_dp_list)
+
+
+  def executeMCPOnlyTestSteps(self, iteration_content, domain_proxy_objects, cbsd_as_dp_list):
+    """Executes the steps from Step23 to Step30 for MCP testcase
+
+    Args:
+       iteration_content: A dictionary with multiple key-value pairs that contain iteration data
+       domain_proxy_objects: Domain proxy objects
+       cbsd_as_dp_list: list of Domain Proxy objects, which act like individual CBSD
+    """
+    cbsds_with_domain_proxies = []
```

Comment:
There is another variable called ```cbsds_with_domain_proxies``` that exists higher up. It looks like this is the same kind of data, just after DPA activation. Please change to something like ```cbsds_with_domain_proxies_after_dpa_activation``` so it's clear this is not the same variable.

Reply:
Yes, variable is used for same kind of data, after above changes this variable is no longer required, modified as per the latest changes